hs/src/Reach/Linearize.hs

```diff
@@ -368,7 +368,7 @@ df_con = \case
           case r of
             Nothing -> return $ Nothing
             Just _ -> do
-              dv <- allocVar $ DLVar at (show fv) (fluidVarType fv)
+              dv <- allocVar $ DLVar at (Just (srcloc_builtin, show fv)) (fluidVarType fv)
```

Comment:
I think it should be `Nothing`, because the user never wrote `show fv`... but then again, we want to know that `v8` is the balance... so maybe it could be `balance()` or `lastConsensusTime()`?